tests/integration/buildcmd/build_integ_base.py

```diff
@@ -381,7 +381,8 @@ def _test_with_go(self, runtime, code_uri, mode, relative_path, architecture=Non
         )
 
         expected = "{'message': 'Hello World'}"
-        if not SKIP_DOCKER_TESTS:
+        if not SKIP_DOCKER_TESTS and architecture == X86_64:
```

Comment:
Doesn't it make sense to remove the `arm64` values from the `parameterized.expand` (inside `TestBuildCommand_Go_Modules_With_Specified_Architecture` in `test_build_cmd.py`) instead of adding this `if` ?

Reply:
I guess Go is weird that it's not supported in ARM but you can still build a project. I think I just fully realized this. It makes sense.